src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Solution.cs

```diff
@@ -67,7 +68,7 @@ public Solution()
 
             Contacts = contactResultList.Select(c => new Contact(c));
             PublishedStatus = solutionResult.PublishedStatus;
-            FrameworkShortNames = solutionResult.FrameworkShortNames;
+            Frameworks = solutionFrameworkResult.Select(x => x.FrameworkName);
```

Comment:
We normally try to avoid using `x` for single letter lambda parameters and to use a letter that relates to the type. I would probably use `f` here (for `framework`).

Reply:
Updated Lambda expression parameter to to use `f` (for `framework`)